CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 * [ENHANCEMENT] Memberlist: client can now keep a size-bounded buffer with sent and received messages and display them in the admin UI (/memberlist) for troubleshooting. #3581 #3602
 * [BUGFIX] Allow `-querier.max-query-lookback` use `y|w|d` suffix like deprecated `-store.max-look-back-period`. #3598
 * [BUGFIX] Memberlist: Entry in the ring should now not appear again after using "Forget" feature (unless it's still heartbeating). #3603
+* [ENHANCEMENT] New /overrides endpoint that returns the limit overrides configuration in YAML format
```

Comment:
Enhancements go before BUGFIX. Also please add PR number to the end.

Reply:
Done